spec/swagger.yaml

```diff
@@ -2306,20 +2307,15 @@ paths:
                           description: General Issue
                         - issueSubTypeCode: REVIEW_TRANSLATION
                           description: Revieanslation
-
-        400: *Error400ResponseDefinition
-        401: *Error401ResponseDefinition
-        429: *Error429ResponseDefinition
-        500: *Error500ResponseDefinition
-  /issues-api/v2/dictionary/issue-states:
+  '/issues-api/v2/dictionary/issue-states':
     get:
-      summary: List all issue states
+      summary: Get Issue States
       description: >-
         Currently the only allowed issue states are `OPENED` and `RESOLVED` but
         Smartling may introduce additional states in future. This request
         returns all current allowed states.
       tags:
-        - Issues
+        - Dictionaries
```

Comment:
I believe this is how it groups the API's, so it should still be under Issues i believe.